services/portal/values-int.yaml

```diff
@@ -47,7 +47,7 @@ firefly:
     runAsGroup: 102
 
   volumes:
-    firefly_workarea_hostpath: /sui/firefly/workarea
+    firefly_shared_workarea_hostpath: /sui/firefly/workarea
```

Comment:
This instance of `workarea` on the RHS is part of the "real" GPFS pathname of the filesystem - something that already physically exists on GPFS at NCSA and therefore (at least for now) should not be changed.  Is that correct?

Same for `values-stable.yaml` below.

Reply:
That is correct.  We would need sysadmin assistance at NCSA to change it...so let's not, at the moment.